### ISSUE TITLE
Pylint Inline Comment - Disabling statement doesn't work

### ISSUE THREAD

**Issue by marcus-tmamtv** [spyder-ide/spyder#7879]
### What steps will reproduce the problem?

FileName: issue_sample.py

FileContent:
--------------------

```
# pylint: disable=E501

print('lorem ipsum Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.')
```
--------------------

Code Analysis is activated in the global spyder settings.
I want now disable some linter options like "line to long" ...
I have added in the first line a pylint settings comment, but it doesn't work.
The editor is still showing the warning symbol, in the case of too long rows.

Is this a config/syntax issue from myside or an general defect/bug...

Please help!

thanks
Marcus

## Versions

* Spyder version: 3.3.1 
* Python version: 3.6.5
* Qt version: 5.9.4
* PyQt5 version: 5.9.2
* Operating System: Darwin 17.7.0

### Dependencies

```
pyflakes >=0.6.0  :  1.6.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.7.4 (OK)
rope >=0.9.4      :  0.10.7 (OK)
jedi >=0.9.0      :  0.12.0 (OK)
psutil >=0.3      :  5.4.5 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
pandas >=0.13.1   :  0.23.0 (OK)
numpy >=1.7       :  1.14.3 (OK)
sympy >=0.7.3     :  1.1.1 (OK)
cython >=0.21     :  0.28.2 (OK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0     :  6.4.0 (OK)
matplotlib >=2.0.0:  2.2.2 (OK)
pylint >=0.25     :  1.8.4 (OK)
```

**Comment by ccordoba12** [comment]
Thanks for reporting. We'll take a look at this in a future release.

**Comment by jnsebgosselin** [comment]
@ccordoba12 Note that I can reproduce this in master also.

![image](https://user-images.githubusercontent.com/10170372/45386142-97454080-b5e0-11e8-964d-e9a4834530c6.png)

**Comment by ccordoba12** [comment]
Then let's leave it for Spyder 4.

**Comment by igurin-invn** [comment]
We're on Spyder 5 now. Still not fixed?

**Comment by dalthviz** [comment]
Checking this you need to use something like `# noqa: E501` in front of the line. Also an option to ignore errors or warnings is available from the preferences:

![image](https://user-images.githubusercontent.com/16781833/123142296-8f857c80-d459-11eb-89de-e17ebb46cf5a.png)

**Comment by dalthviz** [comment]
@ccordoba12 maybe this could be closed?

**Comment by igurin-invn** [comment]
Your example works for me (E501), but "something like" it doesn't.

![image](https://user-images.githubusercontent.com/53919701/123154618-f6844080-d41b-11eb-8a7b-0aab4ddca742.png)
![image](https://user-images.githubusercontent.com/53919701/123154661-069c2000-d41c-11eb-851b-077425f44089.png)

**Comment by ccordoba12** [comment]
There are two things here:

1. We don't use Pylint to lint our files in real time but Pyflakes, so using Pylint messages has no effect.
2. Pylint is used in our `Code Analysis` pane, where comments like `# pylint: disable=E501` do have effect.
3. To ignore linting messages we plan to use `# noqa` standard or `# analysis:ignore` (which was supported in Spyder 3).

Point 3 was reported on issue #11033, so I'm going to close this one as duplicate of that. And we plan to fix #11033 in 5.1.0, to be released at the end of July.
